### ISSUE TITLE
Add support for NVMe devices

### ISSUE THREAD

**Issue by Subetov** [cloudandheat/prometheus_smart_exporter#1]
https://github.com/CloudAndHeat/prometheus_smart_exporter/blob/develop/smart_exporter_helper/__init__.py#L126

Could you add support for NVMe devices please ?
There is no nvme devices on this bus "/sys/bus/scsi/devices/".

**Comment by jangrewe** [comment]
Yes, i'd love to see this! The "Generic NVMe" device config from `check_smart_attributes` works fine, just the exporter doesn't support it. :-(

**Comment by SPFZ** [comment]
We are currently using smartmontools 6.4 therefore we do not have nvme support. If you like to have it please prepare a PR - thanks
